e2e/cypress/support/api/preference.js

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// *****************************************************************************
+// Preferences
+// https://api.mattermost.com/#tag/preferences
+// *****************************************************************************
+
+import theme from '../fixtures/theme.json';
```

Comment:
The path is incorrect. It should be `../../fixtures/theme.json`. Unable to run the tests because of it.

Reply:
Updated